src/api/handlers/channel.ts

```diff
@@ -122,6 +122,15 @@ export async function getPins(channelID: string) {
   return Promise.all(result.map((res) => structures.createMessage(res)));
 }
 
+/** 
+ * Trigger a typing indicator for the specified channel. Generally bots should **NOT** implement this route. 
+ * However, if a bot is responding to a command and expects the computation to take a few seconds, 
+ * this endpoint may be called to let the user know that the bot is processing their message.
+ */
+export function triggerTypingIndicator(channelID: string) {
```

Comment:
```suggestion
export function startTyping(channelID: string) {
```

Reply:
my only gripe with this is the name

Reply:
Discord automatically stops typing indicator after some time. So imo start is not a good name.

Reply:
`Start` does not imply "forever" or "never-ending", it just means begin something with indefinite time to end,